src/services/ledger/paginated-ledger.ts

```diff
@@ -19,21 +19,20 @@ type IFilterQuery = {
   _journal?: Types.ObjectId | string
   start_date?: Date | string | number
   end_date?: Date | string | number
-} & Partial<ILedgerTransaction> & {
-    [key: string]: string[] | number | string | Date | boolean | Types.ObjectId
-  }
+} & Partial<ILedgerTransaction>
 
 type IOptions = {
   session?: ClientSession
 }
 
 export const paginatedLedger = async (
```

Comment:
we generally use 1 argument as an object for multi argument functions in the codebase (ie: `{query, paginationArgs, options}: {query: IFilterQuery, paginationArgs: PaginationArgs, options: IOptions}`)